src/Components/DataHistoryErrorHandlingWrapper/DataHistoryErrorHandlingWrapper.types.ts

```diff
@@ -7,11 +7,15 @@ import {
     ITheme
 } from '@fluentui/react';
 import { IComponentError } from '../../Models/Constants/Interfaces';
-import { IllustrationMessageStyles } from '../IllustrationMessage/IllustrationMessage.types';
+import {
+    IllustrationMessageStyles,
+    IllustrationMessageWidth
+} from '../IllustrationMessage/IllustrationMessage.types';
 
 export interface IDataHistoryErrorHandlingWrapperProps {
     error: IComponentError;
     imgHeight?: number;
+    messageWidth?: IllustrationMessageWidth;
```

Comment:
why do we need this? Can't the message be different lengths based on teh language?

Reply:
this is the enum for the width of the message container, either "compact" or "wide". It is originally a prop to the IllustrationMessage component, just exposed it in the parent to pass along.

Reply:
gotcha. ok. makes sense.